README.rst

```diff
@@ -179,11 +179,6 @@ recommended for sake of speed).
 .. _async-timeout: https://pypi.python.org/pypi/async_timeout
 .. _cChardet: https://pypi.python.org/pypi/cchardet
 
-License
```

Comment:
Apparently this can confuse licensee
https://github.com/licensee/licensee/issues/554